docs/usage/ObjectStorage.md

```diff
@@ -1,10 +1,29 @@
 ## Object Storage Setup
 
-CAM components use S3 object storage as temporary scratch space when performing migrations.  This storage can be any object storage that presents an `S3 like` interface.  Currently, we have tested AWS S3, Noobaa, and Minio.  
+CAM components use object storage as temporary scratch space when performing migrations. This document covers the process of setting up compatible Object Storage to use with CAM on the provider of your choice. The Object Storage credentials obtained from this document are needed for *configuring CAM Replication Repositories*.
```

Comment:
nit: `cloud provider` instead of just `on the provider`

Reply:
ACK